src/overlays/actors/ovl_Bg_Lotus/z_bg_lotus.h

```diff
@@ -5,9 +5,15 @@
 
 struct BgLotus;
 
+typedef void (*BgLotusActionFunc)(struct BgLotus*, PlayState*);
+
 typedef struct BgLotus {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x28];
-} BgLotus; // size = 0x16C
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ BgLotusActionFunc actionFunc;
+    /* 0x160 */ f32 unk160;
+    /* 0x164 */ u8 unk164;
+    /* 0x166 */ s16 unk166;
+    /* 0x168 */ s16 unk168;
+} BgLotus; /* size = 0x16C */
```

Comment:
```suggestion
} BgLotus; // size = 0x16C
```
The styling changed on this, it should be reverted